packages/web3-react/src/helpers/ua.ts

```diff
@@ -15,3 +15,8 @@ export const isMobileOrTablet = isMobile || isTablet;
 
 export const isFirefox = browser.name === 'Firefox';
 export const isEdge = browser.name === 'Edge';
+
+export const checkIfBraveBrowser = async (): Promise<boolean> =>
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Maybe ts-expect-error or fix global type definition?

Reply:
Replaced with `ts-expect-error` for now. Going to declare proper types during refactoring later.